src/mtx3/commands/downlink/getHalfHourDemandVari.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * Downlink command to get reactive energy (`A+R+`) in half hours by date.
+ *
+ * @packageDocumentation
+ *
+ * @example
+ * ```js
+ * import * as getHalfHourDemandVari from 'jooby-codec/mtx3/commands/downlink/getHalfHourDemandVari.js';
```

Comment:
same as getHalfHourDemandVare.ts

Reply:
moreover in the IEnergies interface the key is "vari"

Reply:
applied